fastavro/errors.py

```diff
@@ -0,0 +1,24 @@
+from .util import path_string
+
+
+class UnknownType(Exception):
+    def __init__(self, name):
+        super(UnknownType, self).__init__(name)
+        self.name = name
+
+
+class AvroValueError(ValueError):
+    pass
+
+    @classmethod
```

Comment:
To maintain the same interface as normal exceptions: `AvroValueError("some message")` should behave like `TypeError("some message")`, `ValueError("some message")`, etc. I quite dislike when libraries break this expectation.

Agree on the name, though.